2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,30 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+
+const myCoffeeLatte = {
+  coffeeTpe: 'Ethiopian',
+  isOrganic: true,
+  isRoast: true,
+  roastLevel: "medium",
+  shots: 1.5,
+  ingredients: ['Milk', 'Caremel', 'No Sugar'],
+  MakeCoffee: function () {
+    let haveCoffe;
+    if (this.coffeeTpe === 'Ethiopian' && this.isOrganic === true && this.isRoast === true && this.roastLevel === 'medium') {
+      console.log('Injoy your ☕ Caramel Latte 😀')
+    } else {
+      console.log("Not my COFFEE😉");
+    }
+  }
+}
+console.log(myCoffeeLatte) // this is loging the whole obeject and it's properties.
+
+myCoffeeLatte.ingredients.filter(elements => console.log(elements)) // this is to show the ingredients one in each line.
+
+myCoffeeLatte.MakeCoffee(); // calling makeCoffee method and see how it works.
+
+myCoffeeLatte.isOrganic = false; //this is to check if the conditon is working 
+
+myCoffeeLatte.MakeCoffee(); // result after one of the conditions is assign to a false value.
```

Comment:
I love how you've played with this and made it your own, it's really nice and good for your learning. Detailed comments are brillant also!

Reply:
loved that.😀